vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectDetailsComponent.java

```diff
@@ -228,38 +218,22 @@ public Handler(ProjectInformationService projectInformationService,
       setupExperimentalDesignSearch();
     }
 
-    public void projectId(String projectId) {
-      parseProjectId(projectId);
-      try {
-        projectInformationService.find(projectId).ifPresentOrElse(this::loadProjectData,
-            this::emptyAction);
-      } catch (AccessDeniedException accessDeniedException) {
-        log.error("Access denied when loading project details for project id " + projectId,
-            accessDeniedException);
-        emptyAction();
-      }
+    public void setProjectId(String projectId) {
+      projectInformationService.find(projectId)
+          .ifPresentOrElse(this::loadProjectData, this::emptyAction);
```

Comment:
Because it's currently not clear what should happen if a project/experiment/sample etc. can't be accessed due to user privileges. It's also currently not specified what should happen in the emptyAction().
Therefore i'd rather remove a singular solution and implement this for all components once we decide on what should happen here